osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -224,6 +225,7 @@ public enum OsuConfig
         ConfirmExit,
         AutoSendNowPlaying,
         CursorSize,

```

Comment:
let's rename this to `MenuCursorSize`, so if/when we migrate from osu-stable configs we don't overlook this and have people's settings migrating to the wrong place.